synapse/cortex.py

```diff
@@ -3054,9 +3054,18 @@ async def _onload():
 
                 logextra = await self.getLogExtra(pkg=name, vers=pkgvers)
 
-                if inits is not None:
-                    varname = inits['key']
-                    curvers = await self.getStormPkgVar(name, varname, default=-1)
+                verskey = 'storage:version'
+
+                if inits is None:
+                    await self.setStormPkgVar(name, verskey, -1)
```

Comment:
Does this have to go into onload, or could we just initialize it and then still only fire the task if there are inits/onloads?